internal/generator/backend/template/zkpschemes/groth16/groth16.verify.go.tmpl

```diff
@@ -170,8 +171,18 @@ func (vk *VerifyingKey) ExportSolidity(w io.Writer) error {
 	vk.G2.Gamma, vk.G2.gammaNeg = vk.G2.gammaNeg, vk.G2.Gamma
 	vk.G2.Delta, vk.G2.deltaNeg = vk.G2.deltaNeg, vk.G2.Delta
 
+	cfg, err := solidity.NewExportConfig(exportOpts...)
+	if err != nil {
+		return err
+	}
+
+	vkToExport := VerifyingKeyToExport{
```

Comment:
no need to create a public struct here that will pollute the package api (`VerifyingKeyToExport`) ; just an anonymous struct here is cleaner:
[example here](https://github.com/Consensys/gnark-crypto/blob/564b6f724c3beac52d805e6e600d0a1fda9770b5/internal/generator/tower/asm/amd64/e2_bn254.go#L328)